frog/imports/api/remoteGraphs.js

```diff
@@ -0,0 +1,43 @@
+import { uuid } from 'frog-utils';
+import { Graphs } from '/imports/api/graphs';
+import { graphToString, doImportGraph } from '../ui/GraphEditor/utils/export';
+
+export const removeGraph = (id: string) =>
+  fetch('http://icchilisrv4.epfl.ch:5000/graphs?uuid=eq.'.concat(id), {
+    method: 'DELETE'
+  });
+
+export const collectGraphs = () =>
+  fetch('http://icchilisrv4.epfl.ch:5000/graphs').then(e => e.json());
+
+export const sendGraph = (state: Object, props: Object) => {
+  const newId = uuid();
+  const grph = {
+    title: state.title,
+    description: state.description,
+    tags: '{'.concat(state.tags.join(',')).concat('}'),
+    parent_id: Graphs.findOne(props.graphId).parentId,
+    uuid: newId,
+    graph: graphToString(props.graphId)
+  };
+  fetch('http://icchilisrv4.epfl.ch:5000/graphs', {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json'
+    },
+    body: JSON.stringify(grph)
```

Comment:
are you sure you need to stringify the body? Did you try without? If you need to, that's fine.